build/generate-tray-icons.ts

```diff
@@ -28,23 +28,23 @@ console.log("Generating tray icon pngs");
 
 ensureDirSync(outputFolder);
 
-Promise.allSettled([
-  sharp(Buffer.from(darkTemplate))
+Promise.all([
+  sharp(Buffer.from(lightTemplate))
     .resize({ width: size, height: size })
     .png()
     .toFile(path.join(outputFolder, "trayIconDarkTemplate.png")),
```

Comment:
looks a little odd for `trayIconDarkTemplate.png` to be created from `lightTemplate`. Just rename the variables?

Reply:
No that is on purpose. Because the "Dark" template is used during dark mode, so the template needs to be light. IIRC this is here because of windows, but I might be wrong.